app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -331,6 +331,16 @@ class SampleView extends React.Component {
         issueType = "warning";
         link = `/samples/${sample.id}/results_folder`;
         break;
+      case "BROKEN_PAIRS":
+        status = "COMPLETE - ISSUE";
+        message =
+          "Sorry, something was wrong with your input files. Either paired reads did not follow the same order in both files, " +
+          "or they followed an unconventional naming scheme.";
+        linkText =
+          "Please ensure mates have the same identifier and follow the same order in both input files, then reupload";
```

Comment:
change "mates" to "paired reads" for consistency

Reply:
ok done